commit 46c947af7ead0a939fbd7a93c370e7ead2128d07
Author: serge-sans-paille <sguelton@redhat.com>
Date:   Wed Aug 18 23:48:14 2021 +0200

    Have lit preserve SOURCE_DATE_EPOCH
    
    This environment variable has been standardized for reproducible builds. Setting
    it can help to have reproducible tests too, so keep it as part of the testing
    env when set.
    
    See https://reproducible-builds.org/docs/source-date-epoch/
    
    Differential Revision: https://reviews.llvm.org/D108332

diff --git a/llvm/utils/lit/lit/TestingConfig.py b/llvm/utils/lit/lit/TestingConfig.py
index d534d895e4ba..37558bd5059c 100644
--- a/llvm/utils/lit/lit/TestingConfig.py
+++ b/llvm/utils/lit/lit/TestingConfig.py
@@ -28,7 +28,7 @@ class TestingConfig(object):
                      'TMPDIR', 'TMP', 'TEMP', 'TEMPDIR', 'AVRLIT_BOARD',
                      'AVRLIT_PORT', 'FILECHECK_OPTS', 'VCINSTALLDIR',
                      'VCToolsinstallDir', 'VSINSTALLDIR', 'WindowsSdkDir',
-                     'WindowsSDKLibVersion']
+                     'WindowsSDKLibVersion', 'SOURCE_DATE_EPOCH']
 
         if sys.platform == 'win32':
             pass_vars.append('INCLUDE')
